app/src/main/java/org/jellyfin/androidtv/preference/UserPreferences.kt

```diff
@@ -111,6 +111,11 @@ class UserPreferences(context: Context) : SharedPreferenceStore(
 		 */
 		var audioBehaviour = enumPreference("audio_behavior", defaultAudioBehavior)
 
+		/**
+		 * Preferred behavior for audio streaming.
+		 */
+		var audioNightMode = enumPreference("audio_night_mode", false)
```

Comment:
Will this be an exoplayer only feature or will "night mode" also work with libVLC?

Reply:
Whoops, commented on the wrong line